src/main/java/xyz/iwolfking/woldsvaults/mixins/vaulthunters/skills/MixinModAbilityLabelBindings.java

```diff
@@ -44,6 +45,11 @@ private static void registerDescriptions(CallbackInfo ci) {
                 ability -> AbilityLabelFormatters.ticks(ability.getDurationTicks())
 
         ));
+        ModAbilityLabelBindings.register(ChainMinerAbility.class, Map.of(
+                "blocks",
+                ability -> AbilityLabelFormatters.integer(ability.getUnmodifiedBlockLimit()),
+                "distance",
+                ability -> AbilityLabelFormatters.integer(ability.getRange())
```

Comment:
chainminer's bindings need to be closed with a `));`
i'll rebase to 15-dev to hopefully avoid conflicting with *my own PRs*